prow/apis/prowjobs/v1/types.go

```diff
@@ -495,6 +497,10 @@ func (r Refs) String() string {
 	return strings.Join(rs, ",")
 }
 
+type JenkinsJob struct {
+	GithubBasedProject bool `json:"github_based_project,omitempty"`
```

Comment:
Can we get GoDoc here to explain how this works and link to the plugins that it supports

Reply:
Added godoc but do tell me if it doesn't make sense/needs to be improved, please.